test/cluster/blue-green-deployment/deploy.td

```diff
@@ -0,0 +1,88 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+> CREATE SCHEMA prod_deploy
+> CREATE SOURCE prod_deploy.counter IN CLUSTER prod_deploy FROM LOAD GENERATOR counter (TICK INTERVAL '1s')
```

Comment:
You can use a sub-second TICK INTERVAL here, like 0.01s, to make sure stuff actively happens in the system all the time while the swap is in progress.

Reply:
I tried this before and it always led to the pending dataflows never becoming considered ready, since their lag would stay at > 1 s. I'm not sure if this is a limitation of Mz or the way that we check the pending dataflows.